geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/ingest/SimpleRasterIngest.scala

```diff
@@ -31,16 +31,18 @@ import org.joda.time.{DateTime, DateTimeZone}
 import org.locationtech.geomesa.core.index.DecodedIndex
 import org.locationtech.geomesa.raster.data.{Raster, AccumuloCoverageStore}
 import org.locationtech.geomesa.raster.util.RasterUtils.IngestRasterParams
+import org.locationtech.geomesa.tools.Utils.Formats._
 import org.locationtech.geomesa.utils.geohash.BoundingBox
 
+import scala.collection.parallel.ForkJoinTaskSupport
 import scala.util.Try
 
 object SimpleRasterIngest {
 
   def getReader(imageFile: File, imageType: String): AbstractGridCoverage2DReader = {
     imageType match {
-      case "tiff" => getTiffReader(imageFile)
-      case "dted" => getDtedReader(imageFile)
+      case TIFF => getTiffReader(imageFile)
```

Comment:
This case match fails when files end with extension ".tif", please make this accept other known variations of the same file extension